### PR TITLE
migration guide for third-party services

### PR DESCRIPTION
**Pull request by brainwane**
a step towards #2935

### CODE REVIEW

**Comment by di** [comment]
LGTM, what is this blocked on? BTW you have linting errors.

**Comment by brainwane** [comment]
As I was testing this I realized the copyright date on the documentation was 2015 because that's what was in `__about__.py`, so I updated that as well.

File: warehouse/__about__.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
     __commit__ = None
 
 __author__ = "The Python Packaging Authority"
-__email__ = "donald@stufft.io"
+__email__ = "ernest@python.org"
```
Comment (di) [inline warehouse/__about__.py]: Hmm, maybe we should make this `admin@mail.pypi.org` instead?

Comment (brainwane) [reply warehouse/__about__.py]: Fixed. Ready for re-review.

**Pull request merged**